fe1-web/jest/setup.js

```diff
@@ -13,7 +13,7 @@ jest.mock('@rneui/base/dist/Icon');
 jest.mock('react-blockies', () => 'blockies');
 
 // qr codes are svgs and thus generate huge snapshot files
-jest.mock('core/components/QRCode.tsx', () => 'qrcode');
+jest.mock('core/components/QRCode.tsx', () => 'resources');
```

Comment:
I do not know why this is changed in this PR, but it should stay the same

```suggestion
jest.mock('core/components/QRCode.tsx', () => 'qrcode');
```

Reply:
i changed it as i wanted to refactor the directory name from qrcode to resources. I think the name is more descriptive that way. It might have affected other locations in the fe1 code though

Reply:
The change has not to do with the QR Codes location at all, it only changes how the QR codes are rendered during jest tests.
Instead of computing and storing the whole QR code in the snapshots, this small text is used as a placeholder to say "Hey there is some QR Code here". This change just changes the placeholder, thus leading to a fail of all screens that contains QR Codes.

It is indeed more descriptive, but particularly useless since it is a part only used by testing units. If this change should still be committed, then the snapshots should be updated too.

Reply:
Ohh, no problem then i think my IDE did a wrong refactoring. I will make sure to revert that.